modules/code-editor/src/backend/utils_router.ts

```diff
@@ -67,3 +67,19 @@ export const validateFilePayloadMw = (actionType: 'read' | 'write') => async (re
     next(err)
   }
 }
+
+export const validateFileUploadMw = async (req, res, next) => {
+  if (!req.permissions || !req.body) {
+    next(new Error('module.code-editor.error.missingParameters'))
+  }
+
+  if (!req.permissions['root.raw'].write) {
+    next(new Error('module.code-editor.error.lackUploadPermissions'))
+  }
+
+  if (process.env.BP_MODULE_CODE_EDITOR_DISABLE_UPLOAD) {
```

Comment:
The reason for this variable to not be inside the module's config is it because someone could basically override it?

We should document this variable somewhere in this case

Reply:
Yup! Good point for the doc. i'll add that